dedibox-console/account/how-to/resolve-abuse.mdx

```diff
@@ -1,32 +1,64 @@
-    <Message type="note">
-      Refer to [our abuse notice](https://www.scaleway.com/en/abuse-notice/) for further information about Scaleway's abuse policy.
+1. Click your username, located next to “logged in as” in the top right corner of the Dedibox console. Then, select Abuses from the drop-down menu. A list of your abuse tickets and cases displays.
+2. Click Details to view the specifics of the complaint.
+3. Take note of the details provided and proceed with the necessary steps to address and resolve the issue.
+    <Message type="tip">
+      If you need guidance on resolving outgoing DDoS attacks from your server, refer to our documentation [how to prevent outgoing DDoS](https://www.scaleway.com/en/docs/compute/instances/reference-content/preventing-outgoing-ddos/) for assistance.
     </Message>
+    * In case you receive an abuse for hosting illegal content (including copyright), please delete the content from Scaleway’s infrastructure promptly.
+    * For other types of abuse, below are some suggestions to help you correct it:
+      *  Backup all your server data
+      *  Check for recent file changes
+      *  Check log files on your server
+      *  Check your PHP and cgi files for any backdoors
+      *  Check if any software you use is vulnerable, and update them
+      *  Reinstall your server after backup
+      *  Change passwords of your server
+      *  Backup your website and your email data
+      *  Check for recent files changes, security holes in common CMS you may use (Joomla, SPIP, WordPress…)
+      *  Check version in some software you may use on your computer (email client, FTP client)
+      *  Change your FTP and email passwords
+
+   If you still do not know how to correct the problem, you can contact our [outsourcing partners](https://www.scaleway.com/en/dedibox/outsourcing/) or reach out to Scaleway’s abuse team by opening a ticket to the support directly from your console.
+4. The case will be closed at the end of its lifetime by Scaleway’s abuse team, and so will the abuses forming the case.
+   In case you receive further abuses on the same resource within the case lifetime, it means that you did not take the appropriate measures to prevent further abuse from happening.
+   Therefore, please take additional actions and contact our support team if you need assistance.
+   Failure to take appropriate measures to solve the case may result in the lock of your service as per our [Terms of Services](https://www.scaleway.com/en/contracts/) and the legal framework in force.
+   <Message type="note">
+     If you consider that you have taken all the appropriate measures but still receive new abuses within a case, please reach out immediately to the abuse team by opening a ticket. The team will then assess the measures you have taken and guide you to take new measures if needed in order to prevent the lock of your resource.
```

Comment:
```suggestion
     If you consider that you have taken all the appropriate measures but still receive new abuse tickets within a case, please reach out immediately to the abuse team by opening a ticket. The team will then assess the measures you have taken and guide you to take new measures if needed in order to prevent the locking of your resource.
```